### PR TITLE
Adds Whereabouts CNI plugin enhancement to 416 release notes + GA table

### PR DESCRIPTION
**Pull request by stevsmit**
Version(s):
4.16

Issue:
https://issues.redhat.com/browse/OSDOCS-9479

Link to docs preview:
https://76374--ocpdocs-pr.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-16-release-notes.html#overlapping-ip-configuration-multi-tenant-networks-whereabouts

QE review:
- [ ] QE has approved this change.

Additional information:

### CODE REVIEW

**Comment by dougbtv** [comment]
Spot on! :100:

**Review by dougbtv — APPROVED** [review]

File: release_notes/ocp-4-16-release-notes.adoc [hunk 1]
```diff
@@ -870,6 +870,18 @@ For more information about these configuration fields, see xref:../networking/cl
 The Telemetry and the Insights Operator collects telemetry on IPsec connections. For more information, see xref:../support/remote_health_monitoring/showing-data-collected-by-remote-health-monitoring.adoc#showing-data-collected-from-the-cluster_showing-data-collected-by-remote-health-monitoring[Showing data collected by Telemetry].
 
 [id="ocp-4-16-registry_{context}"]
+[id="overlapping-ip-configuration-multi-tenant-networks-whereabouts"]
+==== Overlapping IP configuration for multi-tenant networks with Whereabouts
+
+Previously, it was not possible to configure the same CIDR range twice and to have the Whereabouts CNI plugin assign IP addresses from these ranges independently. This limitation caused issues in multi-tenant environments where different groups might need to select overlapping CIDR ranges.
```
Comment (lahinson) [inline release_notes/ocp-4-16-release-notes.adoc:876]: Does `CIDR` need to be spelled out in its first mention, or is it well known enough to stand on its own?

File: release_notes/ocp-4-16-release-notes.adoc [hunk 2]
```diff
@@ -870,6 +870,18 @@ For more information about these configuration fields, see xref:../networking/cl
 The Telemetry and the Insights Operator collects telemetry on IPsec connections. For more information, see xref:../support/remote_health_monitoring/showing-data-collected-by-remote-health-monitoring.adoc#showing-data-collected-from-the-cluster_showing-data-collected-by-remote-health-monitoring[Showing data collected by Telemetry].
 
 [id="ocp-4-16-registry_{context}"]
+[id="overlapping-ip-configuration-multi-tenant-networks-whereabouts"]
+==== Overlapping IP configuration for multi-tenant networks with Whereabouts
+
+Previously, it was not possible to configure the same CIDR range twice and to have the Whereabouts CNI plugin assign IP addresses from these ranges independently. This limitation caused issues in multi-tenant environments where different groups might need to select overlapping CIDR ranges. 
+
+With this release, the Whereabouts CNI plugin now supports overlapping IP address ranges through the inclusion of a `network_name` parameter. The `network_name` parameter allows administrators to configure the same CIDR range multiple times within separate `NetworkAttachmentDefinitions`, which enables independent IP address assignments for each range.
```
Comment (lahinson) [inline release_notes/ocp-4-16-release-notes.adoc:878]:
```suggestion
With this release, the Whereabouts CNI plugin supports overlapping IP address ranges through the inclusion of a `network_name` parameter. Administrators can use the `network_name` parameter to configure the same CIDR range multiple times within separate `NetworkAttachmentDefinitions`, which enables independent IP address assignments for each range.
```
A couple minor edits to remove redundancy and anthropomorphic language.

File: release_notes/ocp-4-16-release-notes.adoc [hunk 3]
```diff
@@ -870,6 +870,18 @@ For more information about these configuration fields, see xref:../networking/cl
...
+
+This feature also includes enhanced namespace handling and stores `IPPool` custom resources (CRs) in the appropriate namespaces, and supports cross-namespaces when permitted by Multus. This provides greater flexibility and management capabilities in multi-tenant environments.
```
Comment (lahinson) [inline release_notes/ocp-4-16-release-notes.adoc:880]:
```suggestion
This feature also includes enhanced namespace handling, stores `IPPool` custom resources (CRs) in the appropriate namespaces, and supports cross-namespaces when permitted by Multus. These improvements provide greater flexibility and management capabilities in multi-tenant environments.
```
Per [IBM Style](https://www.ibm.com/docs/en/ibm-style?topic=grammar-pronouns), avoid ambiguous pronoun references (e.g., "this" without a noun following it). I replaced "This" with "These improvements". If you think of a better revision, feel free to use something else.

**Review by lahinson — COMMENTED** [review]
Looks good! I made a couple of suggestions for your consideration, but overall, LGTM.

File: release_notes/ocp-4-16-release-notes.adoc (hunk 1 above)
Comment (stevsmit) [reply release_notes/ocp-4-16-release-notes.adoc:876]: I've never seen it spelled it before. I think if you're poking around the networking docs it's "common knowledge"

**Pull request merged**